lib/platform/bitbucket/index.js

```diff
@@ -385,10 +385,19 @@ function addAssignees() {
   return Promise.resolve();
 }
 
-function addReviewers() {
-  // TODO
-  logger.warn('Cannot add reviewers');
-  return Promise.resolve();
+async function addReviewers(prId, reviewers) {
+  logger.debug(`Adding reviewers ${reviewers} to #${prId}`);
+
+  const { title } = await getPr(prId);
+
+  const body = {
+    title,
+    reviewers: reviewers.map(username => ({ username })),
+  };
```

Comment:
I'm a little surprised at this `body` definition. Does this mean that it's mandatory to include the `title` with this update, yet it's not mandatory to use the PR description as well?

Reply:
The requirement of a title and nothing else was surprising to me too. Without the title, Bitbucket will respond with a 400 status code and the following body content:

```json
{
  "type": "error",
  "error": {
    "fields": {"title": ["This field is required."]},
    "message": "title: This field is required."
  }
}
```

Reply:
Thanks for confirming. Can you also confirm that this approach doesn't accidentally *delete* the existing PR description?

Reply:
It does not delete the existing PR description. I had the same concern, but when I tested against our Bitbucket repos it created PRs with good titles and descriptions. No data was lost when adding the reviewers